### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by marieglr**
coucou coucou,

I fixed the nightmareI created with the margins on the body tag, I worked on the form and other styling issues over the website. Many things can still be improved but we are getting there!
The calc thing is to avoid unuseful scrolling on short pages.

### CODE REVIEW

File: public/stylesheets/MPstylesheet.css [hunk 1]
```diff
@@ -151,6 +154,7 @@ light blue: #32C3FF */
 
 .mp-box {
   display: inline-block;
+  height:calc(100vh - 170px);
 }
```
Comment (mpligioso) [inline public/stylesheets/MPstylesheet.css]: fancy technique there!

Comment (mpligioso) [inline public/stylesheets/MPstylesheet.css:158]: fancy technique there!

**Comment by mpligioso** [comment]
there's a merge conflict on the unverified view. I tried to solve the conflict from my side, but there's a divergence according to git - I'm not sure where it comes from? Eventually it looks like you didn't do this work on top of the most recent work (the router stuff). You might be able to solve that by rebasing on master and solving the conflicts - this will probably also force you to force push on the branch.

**Comment by marieglr** [comment]
Yeah, I had not pulled the master branch since I had merge another one on it directly from github, but I realised that in the middle of the work and fixed most stuff, just forgot one class in unverified.hsb :) 
Should I push to heroku ?

**Pull request merged**